### PR TITLE
AP_AHRS: move init of dcm state into variable declarations

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: libraries/AP_AHRS/AP_AHRS_DCM.h [hunk 1]
```diff
@@ -122,8 +113,11 @@ class AP_AHRS_DCM : public AP_AHRS {
     bool pre_arm_check(bool requires_position, char *failure_msg, uint8_t failure_msg_len) const override;
 
 private:
-    float _ki;
-    float _ki_yaw;
+
+    // these are experimentally derived from the simulator
+    // with large drift levels
+    static constexpr float _ki{0.0087f};
```
Comment (khancyr) [inline libraries/AP_AHRS/AP_AHRS_DCM.h:119]: Const in capital?

**Review by khancyr — APPROVED** [review]

File: libraries/AP_AHRS/AP_AHRS_DCM.h (hunk 1 above)
Comment (peterbarker) [reply libraries/AP_AHRS/AP_AHRS_DCM.h:119]: That would mean changing all the places it is used....

**Comment by peterbarker** [comment]
```
Binary           text            data         bss            total
---------------  --------------  -----------  -------------  --------------
antennatracker   8 (+0.0012%)    0 (0.0000%)  0 (0.0000%)    8 (+0.0010%)
arducopter       -72 (-0.0077%)  0 (0.0000%)  0 (0.0000%)    -72 (-0.0068%)
arducopter-heli  -44 (-0.0048%)  0 (0.0000%)  4 (+0.0031%)   -40 (-0.0038%)
ardurover        -36 (-0.0043%)  0 (0.0000%)  4 (+0.0031%)   -32 (-0.0033%)
arduplane        -36 (-0.0037%)  0 (0.0000%)  4 (+0.0031%)   -32 (-0.0029%)
blimp            4 (+0.0006%)    0 (0.0000%)  4 (+0.0031%)   8 (+0.0010%)
ardusub          172 (+0.0213%)  0 (0.0000%)  -4 (-0.0031%)  168 (+0.0179%)
```

**Review by khancyr — APPROVED** [review]
LGTM

**Comment by peterbarker** [comment]
Closing in favour of https://github.com/ArduPilot/ardupilot/pull/18064

**Pull request closed without merging**